rasa/core/test.py

```diff
@@ -481,13 +489,38 @@ def emulate_loop_rejection(partial_tracker: DialogueStateTracker) -> None:
     partial_tracker.update(ActionExecutionRejected(rejected_action_name))
 
 
+def _get_processor_entity_evaluation_result(
+    processor: "MessageProcessor",
+    tracker: DialogueStateTracker,
+    prediction: PolicyPrediction,
+):
+    previous_event = tracker.events[-1]
+    if isinstance(previous_event, UserUttered):
+        entity_predictions = [
+            entity
+            for prediction_event in prediction.events
+            if isinstance(prediction_event, EntitiesAdded)
+            for entity in prediction_event.entities
+        ]
+        entity_targets = previous_event.entities
```

Comment:
I think you're right @joejuzl . It doesn't contain the `policy_entities` yet as we haven't applied the events from the prediction to the tracker yet.